dvc/utils/fs.py

```diff
@@ -63,3 +64,18 @@ def contains_symlink_up_to(path, base_path):
     if os.path.dirname(path) == path:
         return False
     return contains_symlink_up_to(os.path.dirname(path), base_path)
+
+
+def get_parent_dirs_up_to(dir, root_dir):
+    dir = os.path.normpath(dir)
+    root_dir = os.path.normpath(root_dir)
+    if root_dir not in dir:
+        return []
+
+    dirs = []
+    dirs.append(dir)
+    while dir != root_dir:
+        dir, basename = os.path.split(dir)
```

Comment:
if you only need parent, you could use `os.path.dirname(dir)`

Reply:
sorry, some leftover after refactors